pyomeca/processing/rototrans.py

```diff
@@ -4,7 +4,8 @@
 import xarray as xr
 from scipy.optimize import least_squares
 
-from ..angles import Angles
+from pyomeca import Angles
```

Comment:
Idem as next comment (if the answer is no, then you can merge! You will have to do another release so I can update the conda PR)